exec/util/unit.go

```diff
@@ -43,3 +49,26 @@ func FileFromUnitDropin(root string, unit config.Unit, dropin config.UnitDropIn)
 		},
 	}
 }
+
+func MaskUnit(root string, unit config.Unit) error {
+	filename := filepath.Join(root, SystemdUnitsPath(), string(unit.Name))
+	if err := MkdirForFile(filename); err != nil {
+		return err
+	}
+	return os.Symlink("/dev/null", filename)
+}
+
+func EnableUnit(root string, unit config.Unit) error {
```

Comment:
Having root as a parameter makes it very easy to test... I should write tests for these functions.